chrome.py

```diff
@@ -21,28 +27,50 @@ def __init__(self, error):
 def alfred_error(error):
     alfred.write(alfred.xml([ErrorItem(error)]))
 
-def copy_history(profile):
-    if os.path.isfile(HISTORY_CACHE) and time.time() - os.path.getmtime(HISTORY_CACHE) < HISTORY_CACHE_EXPIRY:
-        return HISTORY_CACHE
+def copy_db(name, profile):
+    cache = os.path.join(alfred.work(True), name)
+    if os.path.isfile(cache) and time.time() - os.path.getmtime(cache) < CACHE_EXPIRY:
+        return cache
 
-    history_file = os.path.join(os.path.expanduser(profile), HISTORY_DB)
+    file = os.path.join(os.path.expanduser(profile), name)
```

Comment:
Yeah pretty weird. https://docs.python.org/2/library/functions.html#file

Glad it's gone in python3.